boards/arm/mimxrt1062_fmurt6/board.cmake

```diff
@@ -0,0 +1,15 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright 2023 NXP
+#
+
+board_runner_args(pyocd "--target=mimxrt1060")
+board_runner_args(jlink "--device=MIMXRT1062xxx6A")
+
+if (${CONFIG_BOARD_MIMXRT1062_FMURT6})
```

Comment:
Why guard this with a CONFIG? It seems like this setting will always be required

Reply:
Removed the check as suggested